packages/core-components/.storybook/stories/welcome/welcome.css

```diff
@@ -0,0 +1,26 @@
+.main {
```

Comment:
This isn't necessary, we will be importing this type of content from Markdown -- Storybook provides the default style for this kind of documentation.

Please remove the file

Related thought: https://github.com/okta/odyssey/pull/912/files#r560609756